### PR TITLE
release v0.0.5

### PR DESCRIPTION
**Pull request by lindgrenj6**

### CODE REVIEW

**Comment by lindgrenj6** [comment]
cc @syncrou @mkanoor

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.0.5] - 2020-08-11
+### Added
+- Unieuq Persist-ref for every response worker #18
```
Comment (syncrou) [inline CHANGELOG.md]: Unique

Comment (lindgrenj6) [reply CHANGELOG.md]: It's just a unique spelling!

**Review by syncrou — APPROVED** [review]
LGTM 👍

**Pull request merged**